authzen/authzen-proxy/.gitignore

```diff
@@ -0,0 +1,184 @@
+# Logs
```

Comment:
Hmm I'd leave this out. Looks like a one-size-fits-all .gitignore attempt. I think keeping `node_modules/` would be sufficient?